stonesoup/functions.py

```diff
@@ -566,3 +567,26 @@ def mod_elevation(x):
     elif N == 3:
         x = x - 2.0 * np.pi
     return x
+
+
+def coerce_to_valid_mapping(mapping: Union[List[int], np.ndarray, StateVector]):
```

Comment:
Mappings really shouldn't be `StateVectors`. Should just be sequence of indexes.

Reply:
Agreed. I think there are many issues with mappings in general that need to be thrashed out.

Reply:
I think I have accepted this point now, so will resolve this.